libs/datamodel/core/tests/common/mod.rs

```diff
@@ -453,7 +453,7 @@ pub(crate) fn parse_and_render_error(schema: &str) -> String {
 
 #[track_caller]
 pub(crate) fn assert_valid(schema: &str) {
-    match datamodel::parse_schema_parserdb(schema) {
+    match datamodel::parse_schema(schema) {
```

Comment:
just a note we need to go back to this at some point, `parse_schema` is a lot more expensive / slower